tests/test_get_flagged_lines.py

```diff
@@ -1,13 +1,20 @@
-import pytest_check as check
-from pii_check.pii_check_hook import get_flagged_lines
+import pytest
 
+from pii_check.pii_check_hook import get_ignored_lines
 
-def test_get_flagged_lines():
-    files = [
-        "tests/test_data/dir_with_files/file_with_pii.txt", "tests/test_data/dir_with_files/file_without_pii.txt",
-        "tests/test_data/dir_with_files/file_with_pii_flag_on", "tests/test_data/dir_with_files/file_with_pii_flag_off",
-        "tests/test_data/dir_with_files/file_with_pii_flag", "tests/test_data/symlink_of_dir_with_files"
-    ]
-    res = get_flagged_lines(files)
-    check.equal(res, [(1, 6, 'tests/test_data/dir_with_files/file_with_pii_flag')])
 
+@pytest.mark.parametrize(
+    "filename, expected",
+    [
+        ("tests/test_data/dir_with_files/file_with_pii.txt", set()),
+        ("tests/test_data/dir_with_files/file_without_pii.txt", set()),
+        ("tests/test_data/dir_with_files/file_with_pii_flag", {1, 2, 3, 4, 5, 6}),
+        ("tests/test_data/dir_with_files/file_with_mismatch_pii_flags", set()),
+    ],
+)
+def test_get_flagged_lines(filename, expected):
```

Comment:
```suggestion
def test_get_ignored_lines(filename, expected):
```

Can also update the filename. Should we use a more generic filename to accommodate more tests?

Reply:
Yes. That's a good idea. Let me do that.